### PR TITLE
BRS-327 Beta landing content update and style fixes

### PR DESCRIPTION
**Pull request by BcGovNeal**
### Jira Ticket:
BRS-BRS-327

### Jira Ticket URL:
https://bcparksdigital.atlassian.net/browse/BRS-BRS-327

### Description:
- Added latest content updates
- Fixed some style issues to match the design

### CODE REVIEW

File: src/static-landing/index.html [hunk 1]
```diff
@@ -18,68 +18,93 @@
...
+      <p>
+        In 2022, BC Parks is on a mission to ensure your experience of parks – from trip planning, to booking a
+        campsite, to contributing to conservation – is just as enjoyable, seamless, and impactful no matter whether
+        you’re in a park, on your laptop, or on the road in between. To meet this goal, we’re setting out to
+        significantly and continuously improve the services we provide to BC’s parks and the people who visit and value
+        them.
+      </p>
+      <p>
+        Through extensive research with park visitors, and in collaboration with the Exchange Lab (the BC Government’s
+        incubator and accelerator for digital innovations like the Healt h Gateway) and some of #BCTech’s leading
```
Comment (cole) [inline src/static-landing/index.html:49]: *Health

File: src/static-landing/css/style.css [hunk 2]
```diff
@@ -3,13 +3,13 @@ body {
 }
 
 a:link {
-  color: #fff;
-  text-decoration: none;
+  color: #036;
+  text-decoration: underline;
```
Comment (cole) [inline src/static-landing/css/style.css:7]: Nit: Not necessary, underline is the default

**Review by cole — APPROVED** [review]
Looks good, one typo

File: src/static-landing/index.html (hunk 1 above)
Comment (BcGovNeal) [reply src/static-landing/index.html]: Fixed in https://github.com/bcgov/bcparks.ca/pull/249/commits/400e913265603d214b0214e25f0528652893be48

File: src/static-landing/css/style.css (hunk 2 above)
Comment (BcGovNeal) [reply src/static-landing/css/style.css]: Fixed in https://github.com/bcgov/bcparks.ca/pull/249/commits/400e913265603d214b0214e25f0528652893be48

**Review by marklise — APPROVED** [review]

**Pull request merged**